Configurations/10-main.conf

```diff
@@ -762,12 +870,16 @@ my %targets = (
     #### machines where gcc doesn't understand -m32 and -m64
     "linux-elf" => {
         inherit_from     => [ "linux-generic32", asm("x86_elf_asm") ],
+        policy           => 'unknown',
+
         cflags           => add(picker(release => "-fomit-frame-pointer")),
         cppflags         => add("-DL_ENDIAN"),
         bn_ops           => "BN_LLONG",
     },
     "linux-aout" => {
         inherit_from     => [ "BASE_unix", asm("x86_asm") ],
+        policy           => 'unknown',
```

Comment:
Maybe a.out should be pushed into the 'deprecated' bucket?  It's probably not much additional maintenance burden, but are there any current systems for it?